Modules/Microsoft365DSC/DSCResources/MSFT_TeamsMeetingBroadcastConfiguration/MSFT_TeamsMeetingBroadcastConfiguration.psm1

```diff
@@ -266,6 +320,9 @@ function Export-TargetResource
         $params = @{
             Identity   = "Global"
             Credential = $Credential
```

Comment:
Please align the =-signs in the hashtable.

Reply:
Done